### DISCUSSION TITLE
Bugs in version v1.12.3+f23078df1

### DISCUSSION THREAD

**Discussion by bertramg666** [btcpayserver/btcpayserver#5642] [Technical Support]
Hello
We just upgraded to v1.12.3+f23078df1 and noticed the following bugs:

1. we are seeing intermittencies when the checkout is created. We use aMember and BTCPAYSERVER. When the user signup and choose a plan the checkout in btcpayserver is created and currently is showing this error message sometimes: "Payment error: Incorrect Gateway response received!" This error is not from aMember because others webmasters who not use aMember are seeing the same problem (I am seeing this in the telegram group)
3. "Link to a custom CSS stylesheet" for checkout is not working.
4. Dark theme for checkout is not working.

This happened after the upgrade:
<img width="1404" alt="image" src="https://github.com/btcpayserver/btcpayserver/assets/77026405/adf23ff8-9dd0-4d20-acae-aa137a40827f">

Also, the hosting provider said:

It looks like this is happening because your VM is running out of memory. If you go to web panel and select your VM and go to Console Log tab, there are these messages at the bottom of the log:

[8663230.584412] Out of memory: Killed process 2746121 (bitcoind) total-vm:4351716kB, anon-rss:1099112kB, file-rss:0kB, shmem-rss:0kB, UID:999 pgtables:8048kB oom_score_adj:0

**Comment by dennisreimann** [comment]
1. With regards to the memory issues I think there's nothing we can do on our end. You might need to investigate this and provide more detail if simply allocating more memory to the VM doesn't solve the problem. Please reopen this issue with more information in that case.
2. and 3. will be solved with #5628.
